docs/docs/30-how-to-guides/25-argo-cd-integration.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to integrate Kargo Stages with Argo CD Applications.
+sidebar_label: Argo CD Integration
+---
+
+# Argo CD Integration
+
+Kargo integrates seamlessly with Argo CD to facilitate a more streamlined application lifecycle
+management process. While Argo CD helps with deploying Kubernetes objects and synchronizing changes
+in the cluster, Kargo focuses on orchestrating the promotion of these changes through various
+`Stage`s of development, such as from `development` to `testing` and then to `production`.
```

Comment:
```suggestion
Argo CD excels at syncing Kubernetes clusters to desired state stored in
a Git repository, but lacks any concept of a "promotion", i.e. updating
the desired state of some application instance to reflect the validated
state of some other application instance. Kargo was created to orchestrate
such processes. Because such orchestration naturally entails some direct
and indirect integration with Argo CD, this page details certain key
elements of the interactions between the two systems.
```